tests/metrics/README.md

```diff
@@ -4,25 +4,19 @@
 Some tests require you to enable the debug mode of cc-oci-runtime. To enable it, please take a look at:
 https://github.com/01org/cc-oci-runtime#18running-under-docker
 
-To run the metrics, just run:
+
+### Run all metrics tests
+To run the metrics, please use Makefile rule `metrics-tests` on the top level of this repository:
 
 ```bash
-$ cd tests/metrics
-$ ./run_docker_metrics
+$ sudo make metrics-tests
```

Comment:
I've raised #811 to capture the need for `root`.

Best practise says we should run everything with the minimum set of privs and only elevate privs when we need to using `sudo(8)`.

@chavafg - We could land this change and then immediately fix #811, but as a minimum we need to document the steps you've outlined there (including the crucial `-E` flag for `sudo(8)`).